src/Styleguide/Components/Lightbox/Lightbox.tsx

```diff
@@ -0,0 +1,286 @@
+import { color, space } from "@artsy/palette"
+import FadeTransition from "Components/Animation/FadeTransition"
+import { bind, once, throttle } from "lodash"
+import React from "react"
+import ReactDOM from "react-dom"
+import styled from "styled-components"
+import { Box } from "Styleguide/Elements/Box"
+import { Flex } from "Styleguide/Elements/Flex"
+import { CloseButton } from "./CloseButton"
+import { Slider, SliderProps } from "./LightboxSlider"
+
+const KEYBOARD_EVENT = "keyup"
+const ZOOM_PER_CLICK = 1.4
+const HIDE_ZOOM_SLIDER_AFTER = 2500
+
+const DeepZoomContainer = styled.div`
+  position: fixed !important;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1000;
+  background-color: ${color("black100")};
+`
+
+export interface LightboxProps {
```

Comment:
this matches up w/ the relay api

Reply:
Not sure I understand this. The field in our schema don’t have uppercase initials such as `Size`. Can you elaborate?

Reply:
It does actually. 

```graphql
{
  artwork(id: "my-dog-sighs-so-this-is-it") {
   image {
      deep_zoom {
        Image {
          Size {
            Width
            Height
          }
        }
      }
    }
  }
}
```
[Results here](https://metaphysics-staging.artsy.net/?query=%23%20Welcome%20to%20GraphiQL%0A%23%0A%23%20GraphiQL%20is%20an%20in-browser%20tool%20for%20writing%2C%20validating%2C%20and%0A%23%20testing%20GraphQL%20queries.%0A%23%0A%23%20Type%20queries%20into%20this%20side%20of%20the%20screen%2C%20and%20you%20will%20see%20intelligent%0A%23%20typeaheads%20aware%20of%20the%20current%20GraphQL%20type%20schema%20and%20live%20syntax%20and%0A%23%20validation%20errors%20highlighted%20within%20the%20text.%0A%23%0A%23%20GraphQL%20queries%20typically%20start%20with%20a%20%22%7B%22%20character.%20Lines%20that%20starts%0A%23%20with%20a%20%23%20are%20ignored.%0A%23%0A%23%20An%20example%20GraphQL%20query%20might%20look%20like%3A%0A%23%0A%23%20%20%20%20%20%7B%0A%23%20%20%20%20%20%20%20field(arg%3A%20%22value%22)%20%7B%0A%23%20%20%20%20%20%20%20%20%20subField%0A%23%20%20%20%20%20%20%20%7D%0A%23%20%20%20%20%20%7D%0A%23%0A%23%20Keyboard%20shortcuts%3A%0A%23%0A%23%20%20%20%20%20%20%20Run%20Query%3A%20%20Ctrl-Enter%20(or%20press%20the%20play%20button%20above)%0A%23%0A%23%20%20%20Auto%20Complete%3A%20%20Ctrl-Space%20(or%20just%20start%20typing)%0A%23%0A%0A%7B%0A%20%20artwork(id%3A%20%22my-dog-sighs-so-this-is-it%22)%20%7B%0A%20%20%09image%20%7B%0A%20%20%20%20%20%20deep_zoom%20%7B%0A%20%20%20%20%20%20%20%20Image%20%7B%0A%20%20%20%20%20%20%20%20%20%20Size%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20Width%0A%20%20%20%20%20%20%20%20%20%20%20%20Height%0A%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D)

Reply:
Woah! I have not ever seen fields with capitals in our schema before. I stand corrected 👌